gamestonk_terminal/jupyter/jupyter_controller.py

```diff
@@ -0,0 +1,240 @@
+"""Jupyter Controller Module"""
+__docformat__ = "numpy"
+
+import argparse
+import difflib
+from typing import List, Union
+
+from prompt_toolkit.completion import NestedCompleter
+
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import (
+    get_flair,
+    system_clear,
+)
+from gamestonk_terminal.menu import session
+
+# pylint: disable=too-many-public-methods,import-outside-toplevel
+
+
+class JupyterController:
+    """Resources Controller class"""
+
+    # Command choices
+    CHOICES = [
+        "cls",
+        "home",
+        "h",
+        "?",
+        "help",
+        "q",
+        "quit",
+        "..",
+        "exit",
+        "r",
+        "reset",
+    ]
+
+    CHOICES_COMMANDS = [
+        "reports",
+        "dashboards",
+        "finra",
```

Comment:
You don't want any of those below, only reports and dashboards. And it should be named `CHOICES_MENUS` instead